definitions/features/candlepin.rb

```diff
@@ -7,6 +7,22 @@ class Features::Candlepin < ForemanMaintain::Feature
     end
   end
 
+  def current_version
```

Comment:
As we are now using `check_min_version` method in the check, we can remove `current_version` method as it is not required any more.